src/Resources/Resources.Test/Resources.Test.csproj

```diff
@@ -24,10 +24,25 @@
     <ProjectReference Include="..\Resources\Resources.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="System">
+      <HintPath>System</HintPath>
```

Comment:
> <HintPath>System</HintPath> [](start = 5, length = 28)

Are these valid hint paths?

Reply:
not sure how these changes got here. reverted.